smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -261,6 +295,8 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
     mockChromeApi =
         new MockChromeApi(mockControl, testController.propertyReplacer);
     mockControl.$replayAll();
+
+    createChromeApiProvider();
```

Comment:
Many changes, like this, seem to have leaked from other PRs.

Reply:
fixed